docs/src/content/docs/get-started/url-shortner.mdx

```diff
@@ -0,0 +1,306 @@
+---
+title: "Building a URL Shortner"
+description: "Create a simple URL Shortner using DiceDB Go SDK."
+sidebar:
+  order: 5
+---
+
+This tutorial guides you through creating a URL shortener using DiceDB, a key-value store, with Go. We’ll set up endpoints to generate short URLs and redirect them to the original URLs.
+
+Prerequisites
+
+1. Go installed (at least version 1.18)
+2. DiceDB server running locally
+3. Basic familiarity with Go and DiceDB
+
+## Environment setup
+
+Refer to [DiceDB Installation Guide](https://dicedb.io/get-started/installation/) to get your DiceDB server up and running with a simple Docker command.
+
+## Starting the application server
+
+1. Clone the repository
+   ```bash
+   git clone https://github.com/Prachi-Jamdade/url-shortner-go-dicedb
```

Comment:
Two things,

1. The examples should allow users to have a super minimal setup to try stuff out. Ideally, they should not require much setup so they don't need to clone repositories to get started.
2. We shouldn't be linking personal repositories in the official docs. If you would like to make the code available to the users you can add it under the examples directory of this repo.